docs: add information on the CE/EE bundles (#814)
gravitee-io/gravitee-docs: pages/partial/bundle/single-bundle.adoc

[IMPORTANT]
====

> As of 3.18.0 version, the distinction between Community Edition (CE) and Enterprise Edition (EE) of Gravitee.io {gravitee-product-name} bundle has been removed.

Reviewer: ```suggestion
Version 3.18.0 of the Gravitee platform has introduced a unified, single distribution bundle for all features available in the Community Edition (CE) and the Enterprise Edition (EE) of Gravitee.io {gravitee-product-name}, which used to be distributed as separate bundles. This change allows for a smooth transition from CE to EE functionality, and enables you to trial EE features without the need for a migration or a standalone EE installation.
```

> You can now stop wondering which bundle you need !

Reviewer: ```suggestion
```

> You can now stop wondering which bundle you need !

Reviewer: ```suggestion

```

> You can now stop wondering which bundle you need !

Reviewer: ```suggestion
```

> You can now stop wondering which bundle you need !

Reviewer: ```suggestion

```

> You can now stop wondering which bundle you need !

Reviewer: Remove this line please

> If you are a community user and want to give a try to our enterprise features, just ask for a license, add it in your installation and restart. You no longer need to download a new version of {gravitee-product-name} !

Reviewer: ```suggestion
If you are a Community user and you want to try out Enterprise Edition features, just install the unified bundle, request a license, add it to the installation, and restart. You no longer need to download a new EE version of {gravitee-product-name} !
```
